src/css/profile/mobile/common/progress.less

```diff
@@ -91,18 +91,18 @@ tau-progress {
 }
 
 .ui-progress-circle.ui-progress-circle-large {
-	width: 124 * @unit_base;
-	height: 124 * @unit_base;
+	width: 124 * @px_base;
```

Comment:
I have a question.
What I've seen the size in the Guide for Progress circle(Indeterminated)
- Large progress circle : 60*60
- Medium progress circle : 48*48
- Small progress circle : 24*24.

How do you get the width, height  value?

Reply:
This is the determinate Progress Circle. It has FABs width and height which is 56*56. Probably next I will make indeterminate progress elements, so I will change this parameters